server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/method/EmailSubmissionSetMethod.scala

```diff
@@ -269,7 +269,7 @@ class EmailSubmissionSetMethod @Inject()(serializer: EmailSubmissionSetSerialize
       mail = {
         val mailImpl = MailImpl.builder()
           .name(submissionId.value)
-          .addRecipients(envelope.rcptTo.filter(m => m.parameters.isEmpty).map(_.email).asJava)
+          .addRecipients(envelope.rcptTo.filter(m => m.parameters.isEmpty || !(m.parameters.get.contains(ParameterName.holdFor) || m.parameters.get.contains(ParameterName.holdUntil))).map(_.email).asJava)
```

Comment:
Not solving `validation step above enforcing rcptTo not to have parameters`.
Please write a method with the return type `Try` to validate the rcptTo (and put it above) instead.

BTW this predicate seems unreadable to me (extract to a method instead).

Reply:
> Not solving validation step above enforcing rcptTo not to have parameters.
> Please write a method with the return type Try to validate the rcptTo (and put it above) instead.
>
> BTW this predicate seems unreadable to me (extract to a method instead).

What is the answer ???